lib/manager/gradle/index.js

```diff
@@ -177,17 +177,12 @@ async function executeGradle(config) {
   return true;
 }
 
-function isProjectRootGradle(fileName) {
-  return fileName === 'build.gradle';
-}
-
 function getDockerRenovateGradleCommandLine(localDir) {
   return `docker run --rm -v ${localDir}:${localDir} -w ${localDir} renovate/gradle ${GRADLE_DEPENDENCY_REPORT_COMMAND}`;
 }
 
 module.exports = {
   extractAllPackageFiles,
-  extractPackageFile,
```

Comment:
Now extractPackageFile doesn't need to be part of the public interface for the gradle manager as we are using extractAllPackageFiles